test/e2e/performanceprofile/functests/1_performance/irqbalance.go

```diff
@@ -84,7 +84,13 @@ var _ = Describe("[performance] Checking IRQBalance settings", Ordered, func() {
 
 			By(fmt.Sprintf("verifying the behavior with irqLoadBalancingDisabled=%v isolatedCPUSet=%v", irqLoadBalancingDisabled, isolatedCPUSet))
 
-			verifyNodes := func(expectedBannedCPUs cpuset.CPUSet) error {
+			verifyNodes := func(irqLoadBalancingDisabled bool) error {
```

Comment:
so it seems I broke it in e74e75ce075c81b538bbaef5d03eec39b640a420 ?

Reply:
No, because we already look at the external value even before https://github.com/openshift/cluster-node-tuning-operator/commit/e74e75ce075c81b538bbaef5d03eec39b640a420#diff-decd79950ca0682eefd520e1171ff0d6c0af15779a02cc59fc71c4dbe1012bfaL87